Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -242,7 +242,7 @@ class PlaceDetailsCardView: UIView {
         // Labels will gracefully collapse on nil.
         titleLabel.text = place.name
         categoryLabel.text = PlaceUtilities.getString(forCategories: place.categories.names)
-        updateURLText(place.url)
+        updateURLText(place.googleProvider?.url)
```

Comment:
This PR shouldn't have to touch `PlaceDetailsCardView` or `PlaceDetailsCardViewController` IMO -- if we want to prioritize Google URLs over others, we can set that URL to the place URL.